src/pages/careers/_model/_locations/_locations.ts

```diff
@@ -32,6 +32,7 @@ import SingaporeFlagIcon from 'images/svg/careers/flag_singapore.svg'
 
 export const cyberjaya: LocationsType = {
     name: 'cyberjaya',
+    jobs_name: 'Cyberjaya',
```

Comment:
Hi @konstantinosG-deriv just a suggestion, Do you think we can use a better name for this? Like `location_name`  or `job_location`. 🙏

Reply:
Ok good idea, thanks 
👍